app/services/mailer/transactional_email.rb

```diff
@@ -0,0 +1,46 @@
+require 'mandrill'
+
+class TransactionalEmail
+  extend TransactionalMailers
+
+  attr_reader :mandrill, :template_name, :brand, :to, :merge_params, :message
+
+  module Types
+    WELCOME = 'welcome-email'
+    CHANGE  = 'plan-change'
+    CANCEL  = 'cancel-plan'
+  end
+
+  MANDRILL = Mandrill::API.new(ENV['MANDRILL_API_KEY'])
+
+  def initialize(template_name, brand, merge_params)
+    @mandrill = MANDRILL
+    @template_name = template_name
+    @brand = brand
+    @merge_params = merge_params
+    @to = brand.twitter_admin.email
+    @message = get_message
+  end
+
+  def send
+    begin
+      mandrill.messages.send_template(template_name, [], message)
```

Comment:
You can probably just have
```ruby
MANDRILL.messages.send_template(template_name, [], message)
```
that way we're not taking up memory allocation by creating the instance variable.